Document Git Dependency and Change Wording (#9)
Nerixyz/instagram_private_api: README.md

dependencies:
...

  instagram_private_api: 
    git:

>       url: https://github.com/Nerixyz/instagram_private_api.git # master branch

Reviewer: Maybe we should also show that you can lock the dependency to a branch/tag/commit:
```yaml
ref: 123ab5f # (optional) branch, tag or commit
```

> # Document

Reviewer: ```suggestion
# Documentation
```

> for read document and learn more how about use to this package visit here [instagram-private-api](https://github.com/Nerixyz/instagram_private_api).

Reviewer: ```suggestion
For documentation visit the GitHub repository [instagram-private-api](https://github.com/Nerixyz/instagram_private_api).
```
